src/Actions/main.vala

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env -S vala workbench.vala --pkg libadwaita-1
+
+public void main() {
```

Comment:
You can fix the spacing between brackets by adding a space here, as the Vala formatter checks the surrounding code to format the rest.
```suggestion
public void main () {
```